it/uk/gov/hmrc/soletraderidentificationfrontend/stubs/AuthenticatorStub.scala

```diff
@@ -65,6 +65,6 @@ trait AuthenticatorStub extends WireMockMethods {
   )
 
   val notFoundErrorJson: JsObject = Json.obj(
-    "errors" -> Json.arr("cid_no_nino")
+    "errors" -> Json.arr("CID returned no record")
```

Comment:
```suggestion
    "errors" -> "CID returned no record"
```
To match the parser.